### PR TITLE
Add homework 02 task 03 and test 03

### PR DESCRIPTION
**Pull request by mukhaw**

### CODE REVIEW

File: hm_1/tasks/task_02_03.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    [2, 4],
+]
+"""
+from itertools import product
+from typing import Any, List
+
+
+def combinations(*args: List[Any]) -> List[List]:
+    all_combinations = []
+    [all_combinations.append(list(i)) for i in list(product(*args))]
```
Comment (Cjkjvfnby) [inline hm_1/tasks/task_02_03.py:20]: Can you do it without a product?

File: test/test_02_03.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from hm_1.tasks.task_02_03 import combinations
+
+
+@pytest.fixture()
```
Comment (Cjkjvfnby) [inline test/test_02_03.py:6]: Just inline these fixtures, it will be more readable to see them inside the tests.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hm_1/tasks/task_02_03.py (hunk 1 above)
Comment (mukhaw) [reply hm_1/tasks/task_02_03.py:20]: Yes

File: test/test_02_03.py (hunk 2 above)
Comment (mukhaw) [reply test/test_02_03.py:6]: I didn’t understand  what means „inline fixure “, can you explain it more detail?

Comment (Cjkjvfnby) [reply test/test_02_03.py:6]: Remove this fixture and move its content to the tests

```python
def test_combinations_with_2_lists_with_not_equal_sizes():
    expected_result = [[1, 3], [2, 3]]
    assert combinations([1, 2], [3]) == expected_result
```

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**